data.sql

```diff
@@ -0,0 +1,6 @@
+/* Populate database with sample data. */
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kgs) VALUES (1, 'Agumon', '2020-02-03', 0, 1, 10.23);
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kgs) VALUES (2, 'Gabumon', '2018-11-15', 2, '1', 8);
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kgs) VALUES (3, 'Pikachu', '2021-01-07', 1, '0', 15.04);
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kgs) VALUES (4, 'Devimon', '2017-05-12', 5, '1', 11);
```

Comment:
- [ ] Since `neutered` is a boolean, its value should be either true or false. Kindly rectify this here 👍